components/atoms/Avatar/avatar.tsx

```diff
@@ -100,7 +107,7 @@ const DefaultAvatar = ({
           className={`${className ? className : ""} object-cover`}
           alt={alt ? alt : "Avatar"}
           fill={true}
-          src={avatarURL ? avatarURL : AvatarImage}
+          src={avatarURL}
```

Comment:
Is `AvatarImage` redundant now?

Reply:
Pretty much. It was redundant before the changes as well, since `avatarURL` would always be valid in that line.

Reply:
It was an image of hacktoberfest as well, so we don't really have a use for that anymore